libusb/descriptor.c

```diff
@@ -32,16 +32,16 @@
 
 static inline uint16_t ReadLittleEndian16(const uint8_t p[2])
 {
-	return (uint16_t)p[1] << 8 |
-	       (uint16_t)p[0];
+	return (uint16_t)((uint16_t)p[1] << 8 |
+					  (uint16_t)p[0]);
```

Comment:
Indentation this way is a bit off, I'd rather see:
```suggestion
	return (uint16_t)((uint16_t)p[1] << 8 |
	                  (uint16_t)p[0]);
```

Reply:
Ah, the Xcode project has this file set for tab=4, but from the emacs configuration directives at the top of the file, it should be tab=8 it seems... will fix.

God I wish we could just decide on a clang-format: https://github.com/libusb/libusb/issues/1442

Reply:
Updated Xcode project: https://github.com/libusb/libusb/pull/1543

Reply:
The thing is - you can never guess the "correct" tab size. All my editors/IDEs are using 4 spaces for tab, and GitHub uses 8.

That's exactly why in cases like this we have to use mixed spaces - tabs for the beginning-of-line indentation, and then spaces to align the "content" of the line, like I suggested in my comments.

Reply:
Well, in this cause I could guess because the file starts with:

`/* -*- Mode: C; indent-tabs-mode:t ; c-basic-offset:8 -*- */`